pkgs/by-name/se/serie/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "serie";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "lusingander";
+    repo = "serie";
+    rev = "v${version}";
+    hash = "sha256-81DXu5SSL5TWtnIDBWCWlatRtEL00h4vOnRA0HuTInQ=";
+  };
+
+  cargoHash = "sha256-Vx3UGsWvudW33CFGU8rS+QCANxG7aXBf0zkYJxcUVps=";
+
+  buildInputs = lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [ CoreGraphics ]);
```

Comment:
Sorry, looks like it also needs `AppKit`!

Reply:
I've tested locally and can confirm that should be all it needs, building for me on darwin.